src/CMakeLists.txt

```diff
@@ -199,13 +199,10 @@ add_feature_info(KeePassHTTP WITH_XC_HTTP "Browser integration compatible with C
 add_feature_info(SSHAgent WITH_XC_SSHAGENT "SSH agent integration compatible with KeeAgent")
 add_feature_info(YubiKey WITH_XC_YUBIKEY "YubiKey HMAC-SHA1 challenge-response")
 
-if(WITH_XC_HTTP)
-    add_subdirectory(http)
-    set(keepasshttp_LIB keepasshttp)
-endif()
+add_subdirectory(http)
 if(WITH_XC_NETWORKING)
-    add_subdirectory(http/qhttp)
-    set(keepassxcnetwork_LIB qhttp Qt5::Network)
+  find_package(CURL REQUIRED)
+  set(keepassxcnetwork_LIB curl)
```

Comment:
${CURL_LIBRARIES}
I think you should also remove this line altogether and use ${CURL_LIBRARIES} directly in target_link_libraries. Then we can get rid of ${keepassxchttp_LIB} when we drop KeepassHTTP.

Reply:
Yes, duh good point